thunder/benchmarks/conftest.py

```diff
@@ -0,0 +1,52 @@
+import platform
+import psutil
+from typing import Any
+
+from asvdb import utils, BenchmarkInfo, BenchmarkResult, ASVDb
```

Comment:
Would you add a comment to this file explaining what it does and how it works, and then a comment for each function describing what it does

Reply:
I think asv is an option to **store** benchmark results and is not required to **run** benchmarks. We should have some logic like

```python
try:
  from asvdb import ...
  HAS_ASVDB = True
except ImportError:
  print('asvdb is not imported, results wont be stored with asv')
  HAS_ASVDB = Fasle

# benchmark runs ...

if HAS_ASVDB:
  db = ASVDB(...)
  db.addResult(...)
```